GeofenceDisplay/test/verify/entry.js

```diff
@@ -0,0 +1,96 @@
+// Run: npm run test-web
```

Comment:
this will move to a sharted folder at the top level I think as per our discussion?

Reply:
Possibly - there's an import of the test file at the bottom, up to this point there has only been one file with tests to be run. Might require some troubleshooting to make work as a shared file